validator/main.py

```diff
@@ -182,6 +195,7 @@ def openai_callable(text: str, topics: List[str]) -> str:
                 api_key, api_base = self.get_client_args()
                 response = OpenAIClient(api_key, api_base).create_chat_completion(
                     model=llm_callable,
+                    response_format={"type": "json_object"},
```

Comment:
Can you update this LLM call so that it generates topics that are constrained to be within `candidate_topics`?

The way to do this can be by:
1. Creating a JSONSchema in the `__init__` method that expects JSON in the following format:
   ```
   {
        "topic_1": {
            "present": True,
            "confidence": 3 (from 1-5)
        },
        ... (for all topics in candidate_topics)
   }
   ```
2. Using that JSONSchema w/ function calling in the `chatcompletion` call.